### PR TITLE
typo: Distiguished -> Distinguished

### PR DESCRIPTION
**Pull request by nschonni**
Submitted separately because it may be product issue since it's in `AttributeName` value

### CODE REVIEW

File: xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml [hunk 1]
```diff
@@ -102,7 +102,7 @@
       </AssemblyInfo>
       <Attributes>
         <Attribute>
-          <AttributeName>System.Configuration.ConfigurationProperty("DistiguishedName", DefaultValue=Mono.Cecil.CustomAttributeArgument)</AttributeName>
+          <AttributeName>System.Configuration.ConfigurationProperty("DistinguishedName", DefaultValue=Mono.Cecil.CustomAttributeArgument)</AttributeName>
```
Comment (rpetrusha) [inline xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml]: This is the literal string in source code, so it should be left as is.

File: xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml [hunk 2]
```diff
@@ -115,7 +115,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.Workflow.Activities.Configuration.ActiveDirectoryRoleFactoryConfiguration.DistinguishedName%2A> is a <xref:System.Configuration.ConfigurationProperty> defined for this type with default value "`DistiguishedName`". This type represents a configuration property, attribute, or child element that is contained in an element of a configuration file.  
+ <xref:System.Workflow.Activities.Configuration.ActiveDirectoryRoleFactoryConfiguration.DistinguishedName%2A> is a <xref:System.Configuration.ConfigurationProperty> defined for this type with default value "`DistinguishedName`". This type represents a configuration property, attribute, or child element that is contained in an element of a configuration file.
```
Comment (rpetrusha) [inline xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml]:
```suggestion
 <xref:System.Workflow.Activities.Configuration.ActiveDirectoryRoleFactoryConfiguration.DistinguishedName%2A> is a <xref:System.Configuration.ConfigurationProperty> defined for this type with default value "`distinguishedName`". This type represents a configuration property, attribute, or child element that is contained in an element of a configuration file.  
```
I checked this both in source and by running a test program.

File: xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml (hunk 1 above)
Comment (nschonni) [inline xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml]:
```suggestion
          <AttributeName>System.Configuration.ConfigurationProperty("DistiguishedName", DefaultValue=Mono.Cecil.CustomAttributeArgument)</AttributeName>
```

Comment (nschonni) [reply xml/System.Workflow.Activities.Configuration/ActiveDirectoryRoleFactoryConfiguration.xml]: OK, it's been undone

**Comment by rpetrusha** [comment]
Thanks, @nschonni. I'll merge your PR now.

**Pull request merged**